<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>germanCreditTransferEdit</class>
 <widget class="QWidget" name="germanCreditTransferEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>302</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Beneficiary Name</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="KLineEdit" name="beneficiaryName"/>
   </item>
   <item row="1" column="3">
    <widget class="KLed" name="statusBeneficiaryName">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Account Number</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryAccNum</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Bank Identifier</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryBankCode</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="KLineEdit" name="beneficiaryAccNum"/>
   </item>
   <item row="3" column="1">
    <widget class="KLed" name="statusAccountNumber">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="KLineEdit" name="beneficiaryBankCode"/>
   </item>
   <item row="3" column="3">
    <widget class="KLed" name="statusBankIdentifier">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Bank Name</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryBankName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="KLineEdit" name="beneficiaryBankName">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clickMessage">
      <string>Will be filled in automatically</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="KLed" name="statusBankName">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="kMyMoneyEdit" name="transferValue">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="resetButtonVisibility" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="3">
    <widget class="KLed" name="statusAmount">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Purpose</string>
     </property>
     <property name="buddy">
      <cstring>transferPurpose</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="KLed" name="statusPurpose">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="KMyMoneyTextEdit" name="transferPurpose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::FixedColumnWidth</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>27</number>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>kMyMoneyEdit</class>
   <extends>QFrame</extends>
   <header>kmymoneyedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>kmymoneytextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>beneficiaryName</tabstop>
  <tabstop>beneficiaryAccNum</tabstop>
  <tabstop>beneficiaryBankCode</tabstop>
  <tabstop>transferPurpose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
